helm/hpcc/templates/dali.yaml

```diff
@@ -27,17 +28,34 @@ spec:
                 {{ include "hpcc.configArg" . }}
               ]
 {{ include "hpcc.addSentinelProbes" . | indent 8 }}
-{{- include "hpcc.addSecurityContext" (dict "root" $ "me" .) | indent 8 }}
+{{- include "hpcc.addSecurityContext" $commonCtx | indent 8 }}
 {{- include "hpcc.addResources" (dict "me" .resources) | indent 8 }}
-{{ include "hpcc.addImageAttrs" (dict "root" $ "me" .) | indent 8 }}
+{{ include "hpcc.addImageAttrs" $commonCtx | indent 8 }}
         volumeMounts:
 {{ include "hpcc.addConfigMapVolumeMount" . | indent 8 }}
-{{ include "hpcc.addDaliVolumeMount" $ | indent 8 }}
+{{ include "hpcc.addDaliVolumeMount" $commonCtx | indent 8 }}
 {{ include "hpcc.addSecretVolumeMounts" (dict "root" $ "categories" (list "system" ) ) | indent 8 }}
+{{- range .services -}}
+{{- if not .disabled -}}
+{{- $_ := set . "access" (splitList " " (include "hpcc.getSashaServiceAccess" .)) -}}
+{{ include "hpcc.addSashaContainer" (dict "root" $ "me" .) | indent 6 }}
+        volumeMounts:
+{{- include "hpcc.addSashaVolumeMounts" (dict "root" $ "me" .) | indent 8 }}
+{{- end }}
+{{- end }}
       volumes:
 {{ include "hpcc.addConfigMapVolume" . | indent 6 }}
-{{ include "hpcc.addDaliVolume" $ | indent 6 }}
+{{- range .services -}}
+{{- if not .disabled -}}
+{{- $_ := set . "access" (without (splitList " " (include "hpcc.getSashaServiceAccess" .)) "dalidata") -}}
+{{- include "hpcc.addSashaVolumes" (dict "root" $ "me" .) | indent 6 }}
+{{- end }}
+{{- end }}
+{{ include "hpcc.addDaliVolume" $commonCtx | indent 6 }}
 {{ include "hpcc.addSecretVolumes" (dict "root" $ "categories" (list "system" ) ) | indent 6 }}
+{{- if or (has "data" .access) (has "dalidata" .access) -}}
+{{ include "hpcc.addSecretVolumes" (dict "root" .root "categories" (list "storage" ) ) | indent 6 -}}
```

Comment:
Should really be using a single call with a conditional list.  If everything else is ok, then it can be merged and @shamser fix it in his PR.